avatica-shaded/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <artifactId>kafka-connect-storage-common-parent</artifactId>
+    <groupId>io.confluent</groupId>
+    <version>10.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>kafka-connect-storage-common-avatica-shaded</artifactId>
+  <name>kafka-connect-storage-common-avatica-shaded</name>
+  <description>avatica shaded package to replace jackson dependencies without CVEs</description>
+  <inceptionYear>2020</inceptionYear>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.4</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <artifactSet>
+                <includes>
+                  <include>org.apache.calcite.avatica:*</include>
+                  <include>com.fasterxml.jackson.core:*</include>
+                </includes>
+              </artifactSet>
+              <filters>
+                <filter>
+                  <artifact>org.apache.calcite.avatica:avatica</artifact>
+                  <includes>
+                    <include>**</include>
+                  </includes>
+                  <excludes>
+                    <exclude>META-INF/MANIFEST.MF</exclude>
+                    <exclude>META-INF/LICENSE</exclude>
+                    <exclude>META-INF/NOTICE</exclude>
+                    <exclude>META-INF/services/com.fasterxml.jackson.core*</exclude>
+                    <exclude>META-INF/maven/com.fasterxml.jackson.core/**</exclude>

```

Comment:
Does the existing `avatica`library really contain Jackson classes that aren't in a shaded package?

Reply:
I'm removing all the meta files that contain jackson from the package. These were in the original package.